package.json

```diff
@@ -275,6 +289,7 @@
     "postinstall": "npm run download:wasm",
     "test": "run-s test:*",
     "test:lint": "eslint src --ext ts",
+    "test:lint:fix": "eslint src --ext ts --fix",
```

Comment:
This won't catch the `tests` directory :-D

Reply:
Right, fixed :)